src/util/support.js

```diff
@@ -1,5 +1,36 @@
+import isNativeRegex from 'regex-native-function'; // checks if a function is native using regex
```

Comment:
You may not need this. You can check for native v0 by doing `'HTMLContentElement' in window` and v1 by doing `'HTMLSlotElement' in window`.